INSTALL

```diff
@@ -15,24 +15,16 @@ limitations under the License.
 
 Detailed build instructions are provided for in both HTML
 and APT format in releases.  The APT sources are 
-available in Subversion checkouts.
+available in Git checkouts.
 
 autotools:
 site/building/autotools.html
 src/site/apt/building/autotools.apt
 
-Apache Ant:
-site/building/ant.html
-src/site/apt/building/ant.apt
+CMake:
+site/building/cmake.html
+src/site/apt/building/cmake.apt
 
 Apache Maven 2:
 site/building/maven.html
 src/site/apt/building/maven.apt
-
-Microsoft Visual Studio:
```

Comment:
How about changing that to vcpkg? You did that [at other places](https://github.com/apache/logging-log4cxx/commit/e64b7cf0fbdd2f799cd83a7361d4ceefb75eff84#diff-7f6195436d46cda43818b2013775e6feR117) as well.

Please commit removing [vstudio.apt](https://github.com/apache/logging-log4cxx/blob/master/src/site/apt/building/vstudio.apt) as well.